frontend/packages/component-library/src/cms/skinnyBanner/SkinnyBanner.tsx

```diff
@@ -0,0 +1,103 @@
+import classNames from 'classnames';
+import {ReactNode, HTMLAttributes} from 'react';
+
+import {LinkButton, LinkButtonProps} from '@/button';
+import {Theme} from '@/common/contexts';
+import Image, {ImageProps} from '@/image';
+import {Heading5, BodyTwoText} from '@/typography';
+
+import moduleStyles from './skinnyBanner.module.scss';
+
+export interface SkinnyBannerProps extends HTMLAttributes<HTMLElement> {
+  /** SkinnyBanner heading */
+  heading: string | ReactNode;
+  /** SkinnyBanner description */
+  description?: string | ReactNode;
+  /** SkinnyBanner image */
+  imageProps?: ImageProps;
+  /** SkinnyBanner link */
+  buttonProps?: LinkButtonProps;
+  /** SkinnyBanner custom background color.
+   *  backgroundImageUrl is higher priority then backgroundColor. */
+  backgroundColor?: string;
+  /** SkinnyBanner custom background url */
+  backgroundImageUrl?: string;
+  /** SkinnyBanner theme value.
+   *  If you're using backgroundImageUrl - you should make sure you set correct theme value to SkinnyBanner.
+   *  */
+  'data-theme'?: Theme;
+  /** SkinnyBanner partner prop */
+  partner?: {title: string; logo: ImageProps};
+  /** SkinnyBanner custom className  */
+  className?: string;
+}
+
+/**
+ * ## Production-ready Checklist:
+ *  * (✔) implementation of component approved by design team;
+ *  * (✔) has storybook, covered with stories and documentation;
+ *  * (✔) has tests: test every prop, every state and every interaction that's js related;
+ *  * (see ./__tests__/SkinnyBanner.test.tsx)
+ *  * (✔) passes accessibility checks;
+ *
+ * ### Status: ```Ready for dev```
+ *
+ * Design System: SkinnyBanner Component.
+ * Renders a Skinny Banner/Section which serves as an opening section of a page. There should only be one Skinny Banner
+ * per page at the very top of the page under the header navigation.
+ */
+const SkinnyBanner: React.FC<SkinnyBannerProps> = ({
+  heading,
+  description,
+  partner,
+  imageProps,
+  buttonProps,
+  backgroundColor,
+  backgroundImageUrl,
+  className,
+  ...HTMLAttributes
+}) => (
+  <div
```

Comment:
I think `aside` would make sense for this.
```suggestion
  <aside
```